### DISCUSSION TITLE
Anyway to combine tailwindcss & styled-component?

### DISCUSSION THREAD

**Discussion by KairuiLiu** [tailwindlabs/tailwindcss#10812] [Help] [answered]
I want to use tailwind in react, and custom some style using styled-components. So, I write the following code, but it seems not work
```js
const GlobalStyle = createGlobalStyle`
  @tailwind base;
  @tailwind components;
  @tailwind utilities;
`

const Wrapper = styled.div`
  // some css
`
```

**Comment by andrepichardo** [comment, marked as answer]
Hi @KairuiLiu!

You can check out this package [tailwind-styled-components](https://www.npmjs.com/package/tailwind-styled-components), which lets you create react components as styled components using TailwindCSS with classnames on multiple lines.

Hope that helps! :)

**Reply by KairuiLiu** [reply]
Thanks😘
